src/abstractdataframe/selection.jl

```diff
@@ -256,6 +236,17 @@ julia> select!(df, :, [:c, :b] => (c,b) -> c .+ b .- sum(b)/length(b))
 │ 1   │ 0.841471 │ 4     │ -0.158529    │
 │ 2   │ 0.909297 │ 5     │ 0.909297     │
 │ 3   │ 0.14112  │ 6     │ 1.14112      │
+
+julia> df = DataFrame(a=1:3, b=4:6);
+
+julia> select!(df, names(df) .=> sum .=> [:A, :B]);
```

Comment:
Maybe better show the syntax without specifying output names (or both)? The most common case is probably the one where you want automatic names.

Reply:
I will add both then. I wanted to show that you can also specify target column names this way.